cache-cli/pkg/storage/sftp.go

```diff
@@ -66,8 +68,9 @@ func createSSHClient(options SFTPStorageOptions) (*ssh.Client, error) {
 	config := &ssh.ClientConfig{
 		User: options.Username,
 		Auth: []ssh.AuthMethod{
-			ssh.PublicKeys(privateKey),
+			ssh.PublicKeys(signer),
 		},
+		// #nosec
 		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
```

Comment:
@shiroyasha @bogyo210 I'm not so sure we can do anything about this one. I did it this way because the old CLI also doesn't check host keys: https://github.com/semaphoreci/toolbox/blob/master/install-toolbox#L24.

What do you think about ignoring this warning?

Reply:
I agree with ignoring it.